velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -63,6 +63,10 @@ class SubstraitVeloxExprConverter {
       const ::substrait::Expression::IfThen& substraitIfThen,
       const RowTypePtr& inputType);
 
+  /// Whether null will be returned on cast failure.
```

Comment:
Does this need to be a method? Can it be a function in anonymous namespace in the .cpp file?

Reply:
Yes, making it a function looks better. Fixed in the new commit. Thanks!